channel/src/main/java/com/synopsys/integration/alert/channel/azure/boards/AzureRedirectUtil.java

```diff
@@ -17,6 +17,7 @@
 import com.synopsys.integration.alert.common.rest.AlertRestConstants;
 
 @Component
+// TODO rename this class
```

Comment:
Is this something that is being saved for a future task or can it be done in this PR?

Reply:
It's a bit of scope creep for this PR, but it is something we will want to do eventually. I think it will happen when we move/remove old channel classes.

Reply:
My opinion is that "Util" is not a suffix we ever want to use. "Utils" is used for classes containing common, stateless (i.e static) methods (e.g. StringUtils). Anything that is a Spring component or a stateful method should not be called "Util" or "Utils".